n6001/hardware/ofs_n6001/build/scripts/entry.tcl

```diff
@@ -15,6 +15,11 @@ switch $tcl_platform(platform) {
       set revision_name import
     }
     post_message "Compiling revision $revision_name"
-    qexec "bash build/scripts/run.sh $revision_name"
```

Comment:
Jump into the FIM's syn_top/ location to do the Quartus build; this will make Quartus happy in terms of relative paths to ip/sdc files that are baked into the imported FIM files.